lib/packages.js

```diff
@@ -135,29 +139,39 @@ bedrock.events.on('bedrock-express.configure.static', callback => {
     }
 
     // rebuild `importAll.js` module
-    _buildImportAllModule();
+    _buildImportAllModule(callback);
   } catch(e) {
     logger.error('[bedrock-views] failed to read packages and ' +
       'create "importAll" module.', e);
     return callback(e);
   }
+};
+
+bedrock.events.on('bedrock-express.configure.static', () => {
+  // serve minified js
+  bedrock.config.express.static.push({
+    route: '/system/app/main.min.js',
+    path: bedrock.config['bedrock-webpack'].out,
+    file: true
+  });
```

Comment:
No reason right now.  Moved.  I was thinking it might need to be more complex if we needed to look up names with hashes or similar.